test/unit/directives/centreGender_spec.js

```diff
@@ -18,4 +40,12 @@ describe('UNIT centreGenderDirective', function () {
   xit('should bind an availability property to its template.', function () {
   });
 
+  xdescribe('the `scope` property', function () {
+    it('should only have attribute names in lowercase (otherwise they won\'t work properly)', function () {
+      Object.getOwnPropertyNames(directive.scope).forEach(function () {
+        expect(/[a-z_]/.test(attr), 'attribute named "' + attr + '" is not lowercase');
+      });
+    });
+  });
+
 });
```

Comment:
The project template that was used to speed up the initial setup does not have an easy way to create proper unit tests. It assumes that all the testing should be done in some weird browersified angular-friendly state, rather than simply being able to write basic tests for functional code.